hack/cert-setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
helper script

Reply:
We could. This can be refactored as a separate task if somebody wants to do this.